### PR TITLE
Spanish Castilian translation

### PR DESCRIPTION
**Pull request by SantosAlarcon**
Hi! I'm Santos from Spain!

I would glad to contribute to this project by adding the Spanish Castilian translation, to make understandable for the people of Spain. All the string variables are translated so the user can see the translated strings of the UI in this language.

I send you cheers from Spain.

### CODE REVIEW

File: locales/es-es/translation.json [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    "key": "Clave",
+    "value": "Valor",
+    "addEntry": "Añadir entrada"
+  },
+  "dialogControls": {
+    "save": "Guardar",
+    "cancel": "Cancelar"
+  },
+  "uploadImage": {
+    "uploadInstructions": "Sube una imágen desde tu dispositivo:",
```
Comment (AntennaeVY) [inline locales/es-es/translation.json:13]: Change "imágen" to "imagen"

File: locales/es-es/translation.json [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    "addViaUrlInstructions": "O añade una imágen desde una URL:",
```
Comment (AntennaeVY) [inline locales/es-es/translation.json:14]: Change "imágen" to "imagen"

File: locales/es-es/translation.json [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    "autoCompletePlaceholder": "Selecciona o pega el src de la imágen",
```
Comment (AntennaeVY) [inline locales/es-es/translation.json:15]: Change "imágen" to "imagen"

File: locales/es-es/translation.json [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    "alt": "Alt:",
+    "title": "Título:"
+  },
+  "imageEditor": {
+    "editImage": "Editar image"
```
Comment (AntennaeVY) [inline locales/es-es/translation.json:20]: Change "image" to "imagen"

File: locales/es-es/translation.json [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+    "inlineCode": "Formato de código en línea",
+    "link": "Crear enlace",
+    "richText": "Texto enriquecido",
+    "diffMode": "Modo diferencial",
+    "source": "Modo fuente",
+    "admonition": "Insertar admonición",
+    "codeBlock": "Insertar bloque de código",
+    "editFrontmatter": "Editar frontmatter",
+    "insertFrontmatter": "Insertar frontmatter",
+    "image": "Insertar imágen",
```
Comment (AntennaeVY) [inline locales/es-es/translation.json:78]: Change "imágen" to "imagen"

File: locales/es-es/translation.json [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+    "undo": "Deshacer {{shortcut}}",
+    "redo": "Rehacer {{shortcut}}"
+  },
+  "admonitions": {
+    "note": "Nota",
+    "tip": "Consejo",
+    "danger": "Peligro",
+    "info": "Información",
+    "caution": "Aviso",
+    "changeType": "Seleccionar tipo de adminición",
```
Comment (AntennaeVY) [inline locales/es-es/translation.json:95]: Change "adminición" to "admonición"

Comment (petyosi) [reply locales/es-es/translation.json:95]: @AntennaeVY @SantosAlarcon I am not sure what's wrong or right here...

Comment (AntennaeVY) [reply locales/es-es/translation.json:95]: Corrections to Spanish words spelling

Comment (AntennaeVY) [reply locales/es-es/translation.json:95]: Done [#637](https://github.com/mdx-editor/editor/pull/637)

**Pull request merged**